src/qml.qrc

```diff
@@ -37,6 +37,8 @@
         <file>qml/Mysthea/Components/RoundButton.qml</file>
         <file>qml/Mysthea/Components/TextIconComboBox.qml</file>
         <file>qml/Mysthea/Components/ProjectButton.qml</file>
+        <file>qml/Mysthea/Components/AppContainer.qml</file>
```

Comment:
When you'll move these components inside `style/TheFall/Components/` change these file path.